kit/KitWebSocket.cpp

```diff
@@ -261,23 +261,24 @@ void BgSaveParentWebSocketHandler::reportFailedSave(const std::string &reason)
 
 void BgSaveParentWebSocketHandler::handleMessage(const std::vector<char>& data)
 {
-    LOG_DBG(_socketName << ": recv from parent [" <<
-            COOLProtocol::getAbbreviatedMessage(data));
-
     const StringVector tokens = StringVector::tokenize(data.data(), data.size());
 
-    // Should pass only:
-    // "error:", "forcedtracevent", "unocommandresult:"
-    // "statusindicator[start|finish|setvalue]"
-
-    // Badly don't want modified state coming from the background processx
-    if (tokens[1] == "statechanged:" ||
-        tokens[1] == "calcfunctionlist:")
+    // Only accept messages that make sense
+    if (tokens[1] != "error:" &&
+        tokens[1] != "jsdialog:" &&
+        tokens[1] != "progress:" &&
+        tokens[1] != "tracevent:" &&
```

Comment:
Should these be traceevent: and forcedtraceevent: not tracevent:/forcedtracevent: ? The former is what I think I see in wsd/protocol.txt and other code